package-lock.json

```diff
@@ -15419,9 +15425,9 @@
       "dev": true
     },
     "storybook-chrome-screenshot": {
-      "version": "1.3.0",
```

Comment:
I would expect this to be the only change to the lockfile, I think the rest might have been an accident (especially some with major version bump). Please check it.